BUILD.md

```diff
@@ -63,11 +63,28 @@ Distribution | Requires
 Alpine | cmake, g++, libexecinfo, make
 CentOS 8 | clang, cmake, diffutils, libatomic, libstdc++-static, make, zlib-devel
 Fedora | cmake, gcc-c++, libatomic, libstdc++-static, make
+Raspbian 32-bit | cmake
 Rocky | clang, cmake, diffutils, libatomic, libstdc++-static, make, zlib-devel
 Ubuntu | cmake, g++, make
 
 Note that you only need to run `make libs` once the first time you build (or if the version of LLVM in the `lib/llvm/src` Git submodule changes).
 
+### 32-bit Raspbian
+
+Installing on a 32-bit Raspbian is slightly different from other Linux based
+Operating Systems. There are three important things to note:
+
+- at the moment, only `gcc` can be used to build Pony; `clang` currently doesn't work.
+- you'll need to override the `tune` option to configure.
+- currently, we can't build a release version of the Pony compiler and runtime, only the debug version.
+
+```bash
+make libs
+make configure tune=native config=debug
+make build config=debug
```

Comment:
@kulibali is `tune=native` required for both of these? I know to get it to build it is only needed by configure, but I'm not sure if once you give to configure if it is carried over for future operations. Ditto `arch` etc all.